src/components/Button/index.js

```diff
@@ -28,7 +28,8 @@ class Button extends Component {
   };
 
   render() {
-    const { accessibilityLabel, color, disabled, onPress, testID, title } = this.props;
+    const { accessibilityLabel, color, onPress, testID, title } = this.props;
+    const disabled = this.props.disabled !== undefined ? this.props.disabled : 'disabled' in this.props;
```

Comment:
To match the html spec for boolean attributes. The presence of a boolean attribute results in a true value regardless of what that value is set to, which I now realize means `diabled={false}` should also equal true. This is probably undesirable behavior so I'll just revert this change.